cancer_ai/validator/competition_manager.py

```diff
@@ -128,12 +186,16 @@ async def evaluate(self):
             start_time = time.time()
             y_pred = await model_manager.run(X_test)
             run_time_s = time.time() - start_time
-            print("Model prediction ", y_pred)
-            print("Ground truth: ", y_test)
+            # print("Model prediction ", y_pred)
 
             model_result = competition_handler.get_model_result(
                 y_test, y_pred, run_time_s
             )
+            # log_results_to_wandb(y_test, y_pred, run_time_s, hotkey)
             self.results.append((hotkey, model_result))
+            self.log_results_to_wandb(hotkey, model_result)
 
-        return self.results
+        winning_hotkey = sorted(
+            self.results, key=lambda x: x[1].accuracy, reverse=True
+        )[0][0]
+        return winning_hotkey
```

Comment:
can we make it return also competition id as discussed? rewarded would rely on that

Reply:
check `run_competitions_tick`